platform-service/pyproject.toml

```diff
@@ -14,7 +14,7 @@ dependencies = [
     "cryptography>=41.0.7",
     "unstract-flags @ file:///${PROJECT_ROOT}/../unstract/flags",
     "requests>=2.31.0",
-    "unstract-sdk~=0.54.0rc1",
+    "unstract-sdk==0.54.0rc2",
```

Comment:
@gaya3-zipstack [MINOR] Should we pin the version to rc2?

Reply:
Did a quick test. Even keeping in open to patch version using tilda is ok as it is not pulling rc3 (I was assuming it would as we are allowing patch versions using tilda)